draft-btw-add-home.txt

```diff
@@ -976,9 +976,10 @@ Internet-Draft       Encrypted DNS in Home Networks        November 2020
    server.
 
    Upon discovery of a DoH resolver (Section 4), the DoH client may
-   contact that DoH server to retrieve the list of supported DoH
-   services using the well-known URI.  DoH clients repeats that request
-   regularly to retrieve an updated list of supported DoH services.
+   contact that DoH resolver to retrieve the list of supported DoH
+   services using DEER [I-D.pauly-add-deer].  This will allow the client
+   to discover the resolver's supported DoH templates (or DoH resolvers
+   the discovered resolver designates) using DNS SVCB queries.
```

Comment:
`that the discovered resolver designates`

Reply:
Done